README.md

```diff
@@ -39,7 +39,7 @@ Some Inputs are required, while others are optional.
 
 - `dciClientId`: Remote CI client ID, this is passed as a secret.
 - `dciApiSecret`: Remote CI API secret, this is passed as a secret.
-- `dciTopics`: A comma-separated list of DCI topics example:
+- `dciTopics`: A comma-separated list of DCI topics example, or a wildcard for a particular topic, that uses all the latest versions of that topic.
```

Comment:
Hi @tonyskapunk thank you for the PR! 

I have two questions about how it works:

- Do you allow wildcards like `dciTopics: OCP-4.1*` or by wildcards you mean `all-OCP`? 
- "all the latest versions of that topic" - does that mean all-GAs or all OCPs starting from 4.6?

Reply:
> - Do you allow wildcards like dciTopics: OCP-4.1* or by wildcards you mean all-OCP?

The code allows only "all" of a topic type, `all-OCP`, `all-OSP` or `all-RHEL`. I'll fix the wordking to make this clear.

> -  "all the latest versions of that topic" - does that mean all-GAs or all OCPs starting from 4.6?

The last 6 versions, for OCP at this moment it is 4.16 .. 4.11.  When 4.17 is available then it will go from 4.17 .. 4.12